.github/workflows/password-tool.yml

```diff
@@ -0,0 +1,77 @@
+name: Password tool script tests
+on:
+  pull_request:
+    paths:
+      - 'unattended_installer/passwords_tool/**'
+
+jobs:
+  Build-password-tool-and-wazuh-install-scripts:
+   runs-on: ubuntu-latest
+   steps:
+      - uses: actions/checkout@v2
+      - name: Build password-tool and wazuh-install scripts
+        working-directory: ./unattended_installer
+        run: |
+          bash builder.sh -p -i
+        shell: bash 
+      - uses: actions/upload-artifact@v3
+        with:
+          name: scripts
+          path: | 
+            unattended_installer/wazuh-install.sh
+            unattended_installer/wazuh-passwords-tool.sh
+          if-no-files-found: error 
+  test-password-tool-1:
+    runs-on: ubuntu-latest
+    needs: Build-password-tool-and-wazuh-install-scripts
```

Comment:
Separate jobs. One for Wazuh install, one for test-password-tool-1, and the other for #2.